net-misc/balance/balance-3.57-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs fcaps
+
+DESCRIPTION="TCP Load Balancing Port Forwarder"
+HOMEPAGE="http://www.inlab.de/balance.html"
+SRC_URI="http://www.inlab.de/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="caps"
+
+DEPEND="caps? ( sys-libs/libcap )"
+RDEPEND=""
+
+PATCHES=( "${FILESDIR}"/${P}-Makefile.patch )
+
+src_prepare() {
+	default
+
+	tc-export CC
+}
+
+src_install() {
+	default
+
+	#autocreated on program start, if missing
+	rmdir "${D}"/var/run/${PN}
+}
+
+pkg_postinst() {
+	if use caps ; then
+		fcaps 'CAP_NET_BIND_SERVICE=+eip' "/usr/sbin/balance"
+		/bin/mkdir -m 01777 /var/run/balance/
```

Comment:
This line should not be here. `/var/run` is usually symlink to `/run` which is mounted as `tmpfs`, therefore `/var/run/balance` will not survive restart anyway. Btw, `balance` will probably create it on start, according to the comment in `src_install`.

Reply:
balance creates in on start. Permissions might be resolbed by start-up script that is not provided for this tool, so it's up to end user.